scala/README.md

```diff
@@ -154,27 +165,31 @@ Please read the [Scala Style Guide] carefully. The main points to consider are:
     ```scala
     package foo.bar
 
-    // bad
+    // Bad
     import baz.Qux
 
-    // good
+    // Good
     import foo.bar.baz.Qux
     ```
 
+0. Avoid wildcard-importing entire packages: `import package._`
```

Comment:
do we have a good general rule for this? IMO it's always more useful to not use wildcard imports.